shuup/admin/modules/products/views/edit.py

```diff
@@ -258,4 +259,15 @@ def get_context_data(self, **kwargs):
                     self.object.product, self.request
                 )
 
+        product_validator_provides = sorted(get_provide_objects("admin_product_validator"), key=lambda x: x.order)
```

Comment:
I think the validators don't need to have an order 🤔

Reply:
The reason I added an order(priority) is in case there are more ProductValidators and one of them is generating an issue of type error. As all the issues share the same alert notification, sorting by issue type allows to show the alert with the most relevant level.